pkg/commands/image/push_test.go

```diff
@@ -11,6 +11,7 @@ import (
 
 	"github.com/Khan/genqlient/graphql"
 	"github.com/docker/docker/api/types"
+	dockerImage "github.com/docker/docker/api/types/image"
```

Comment:
This is aliased because we are importing `mass/pkg/commands/image` as well, which obv conflicts.